x-pack/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { defaultsDeep, get } from 'lodash';
+import { getDefaultObject } from './get_default_object';
+// @ts-ignore untyped module
+import { getExportTypesHandler } from './get_export_type_handler';
+import {
+  AggregationBuckets,
+  AggregationResults,
+  JobTypes,
+  KeyCountBucket,
+  RangeAggregationResults,
+  RangeStats,
+  UsageObject,
+} from './types';
+
+const JOB_TYPES_KEY = 'jobTypes';
+const JOB_TYPES_FIELD = 'jobtype';
+const LAYOUT_TYPES_KEY = 'layoutTypes';
+const LAYOUT_TYPES_FIELD = 'meta.layout.keyword';
+const OBJECT_TYPES_KEY = 'objectTypes';
+const OBJECT_TYPES_FIELD = 'meta.objectType.keyword';
+const STATUS_TYPES_KEY = 'statusTypes';
+const STATUS_TYPES_FIELD = 'status';
+
+const DEFAULT_TERMS_SIZE = 10;
```

Comment:
Previously, these numbers were smaller, and exactly the size they need to be. I think it's ok to be a little bit lazy though. Having Elasticsearch reserve 10 buckets for a terms agg instead of 3 (for the export types) won't make a huge difference.